deps/spidershim/test/value.cc

```diff
@@ -1434,3 +1434,177 @@ TEST(SpiderShim, ArrayBuffer_NeuteringApi) {
+                    .ToLocalChecked()
+                    ->NumberValue(context)
+                    .FromJust());
+
+  v8::Local<Value> args4[] = {v8_num(7.7), v8_num(8.8), v8_num(9.9),
+                              v8_num(10.11)};
+  Local<v8::Array> a4 = Local<v8::Array>::Cast(
+      Foo->Call(context, Foo, 4, args4).ToLocalChecked());
+  EXPECT_EQ(4u, a4->Length());
+  EXPECT_EQ(7.7, a4->Get(context, v8::Integer::New(isolate, 0))
+                    .ToLocalChecked()
+                    ->NumberValue(context)
+                    .FromJust());
+  EXPECT_EQ(8.8, a4->Get(context, v8::Integer::New(isolate, 1))
+                    .ToLocalChecked()
+                    ->NumberValue(context)
+                    .FromJust());
+  EXPECT_EQ(9.9, a4->Get(context, v8::Integer::New(isolate, 2))
+                    .ToLocalChecked()
+                    ->NumberValue(context)
+                    .FromJust());
+  EXPECT_EQ(10.11, a4->Get(context, v8::Integer::New(isolate, 3))
+                      .ToLocalChecked()
+                      ->NumberValue(context)
+                      .FromJust());
+
+  Local<v8::Value> r1 =
+      ReturnThisSloppy->Call(context, v8::Undefined(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r1->StrictEquals(context->Global()));
+  Local<v8::Value> r2 =
+      ReturnThisSloppy->Call(context, v8::Null(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r2->StrictEquals(context->Global()));
+  Local<v8::Value> r3 =
+      ReturnThisSloppy->Call(context, v8_num(42), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r3->IsNumberObject());
+  EXPECT_EQ(42.0, r3.As<v8::NumberObject>()->ValueOf());
+  Local<v8::Value> r4 =
+      ReturnThisSloppy->Call(context, v8_str("hello"), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r4->IsStringObject());
+  EXPECT_TRUE(r4.As<v8::StringObject>()->ValueOf()->StrictEquals(v8_str("hello")));
+  Local<v8::Value> r5 =
+      ReturnThisSloppy->Call(context, v8::True(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r5->IsBooleanObject());
+  EXPECT_TRUE(r5.As<v8::BooleanObject>()->ValueOf());
+
+  Local<v8::Value> r6 =
+      ReturnThisStrict->Call(context, v8::Undefined(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r6->IsUndefined());
+  Local<v8::Value> r7 =
+      ReturnThisStrict->Call(context, v8::Null(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r7->IsNull());
+  Local<v8::Value> r8 =
+      ReturnThisStrict->Call(context, v8_num(42), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r8->StrictEquals(v8_num(42)));
+  Local<v8::Value> r9 =
+      ReturnThisStrict->Call(context, v8_str("hello"), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r9->StrictEquals(v8_str("hello")));
+  Local<v8::Value> r10 =
+      ReturnThisStrict->Call(context, v8::True(isolate), 0, NULL)
+          .ToLocalChecked();
+  EXPECT_TRUE(r10->StrictEquals(v8::True(isolate)));
```

Comment:
FWIW, all of these `v8::` prefixes are unnecessary.  :)

(I can take them out after merging too.)